opnode/rollup/driver/state.go

```diff
@@ -237,7 +237,7 @@ func (s *state) createNewL2Block(ctx context.Context) (eth.L1BlockRef, error) {
 		return eth.L1BlockRef{}, nil
 	}
 	// Don't produce blocks until past the L1 genesis
```

Comment:
```suggestion
	// Don't produce blocks until at the rollup-genesis block of the L1 chain
	// note: deposits within this L1 block will be included in L2.
```
cc @trianglesphere for review. There are some interesting edge-cases here with deposits: we have to make sure to include those of the genesis block (and maybe everything before if we have a migration from previous system) are all included in L2. But based on timestamps the sequencer may try to build on the next L1 block before those deposits are processed. And the verifier will also need to do the same.

Reply:
My understanding is that we don't want to actually read the L1 genesis when producing L2 blocks (and start after it). With this I guess it still works, but we have a weird thing where the L2 genesis should maybe point back one block.

I don't actually mind missing deposits prior to the start of the rollup

Reply:
Opened an issue to track the genesis related edge cases here: https://github.com/ethereum-optimism/optimistic-specs/issues/323

Think this PR is fine and we can address the deposit related edge-cases of genesis later.